v3/tutorials/09-cumulus/a-start-relay/index.mdx

```diff
@@ -216,69 +216,6 @@ cargo build --release
 
 If the help page is printed, you have succeeded in building a Cumulus-based parachain collator.
 
-## Parachain node template overview
```

Comment:
Parsed into HTGs in this PR (mostly) 
NOTE: the specific callouts and gotchas included here are now removed for clarity. They may need to be reintroduced if users hit issues.